cmd/ore/aws/upload.go

```diff
@@ -236,6 +239,21 @@ func runUpload(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	tagMap := make(map[string]string)
+	if len(uploadTags) > 0 {
+		for _, tag := range uploadTags {
+			splitTag := strings.SplitN(tag, "=", 2)
+			key, value := splitTag[0], splitTag[1]
+			tagMap[key] = value
+		}
+		err = API.CreateTags([]string{hvmID}, tagMap)
```

Comment:
style nit: I'm typically a fan of the `if err := func(); err != nil {` style but this is also fine. We do typically keep the if next to the err assignment with no line break, however

Reply:
No biggie, but I agree.